app.py

```diff
@@ -77,6 +120,6 @@ def morph(images):
 
 
 if __name__ == '__main__':
-    app.config['SECRET_KEY'] = os.environ.get('SECRET') or 'hyjygjgj'
+    app.config['SECRET_KEY'] = app.config['SECRET_KEY'] = os.environ.get('SECRET') or 'hyjygjgj'
```

Comment:
Why this change?
```suggestion
    app.config['SECRET_KEY'] = os.environ.get('SECRET') or 'hyjygjgj'
```
Is the above line not working?

Reply:
This was a last ditch copy and paste error, `result. app.config['SECRET_KEY'] = os.environ.get('SECRET') or 'hyjygjgj'` would be fine.